vendors/nordic/boards/nrf52840-dk/ports/common_io/iot_test_common_io_internal.c

```diff
@@ -29,6 +29,18 @@
 
 #include "iot_test_common_io_internal.h"
 
+#if defined( IOT_TEST_COMMON_IO_FLASH_SUPPORTED ) && ( IOT_TEST_COMMON_IO_FLASH_SUPPORTED >= 1 )
+/* FLASH */
+    void SET_TEST_IOT_FLASH_CONFIG( int testSet )
+    {
+        /* This value depends on linked flash usage. At time of writing, test project ended @ 0x49bc3
+         * Some tests require this to be page-aligned
+         */
+        ultestIotFlashStartOffset = 0x75000;
```

Comment:
Is this magic number specific to this linker, compiler, device, feature set in the test?
Can we derive this magic number from the linker script?

Reply:
Specific to our `aws_tests` architecture for this board. The Code Regions are "free-game", though our code save  starts at Code Region 0 and ended at 0x49bc3 -- for this PR's `aws_tests`

I agree that a symbol like `__user_flash_start__` and `__user_flash_end__` would be a useful improvement that should be made for all of our boards. Currently there is no such shared mechanism.
I've added a separate task to research how we can address this across all boards.

<p align="center">
<img width="250" height="500" src="https://user-images.githubusercontent.com/59750547/89073260-f7f1d980-d32e-11ea-901a-c54339aa44de.png">
</p>